File: src/main/java/com/openvalue/boardgameratings/service/rating/RateEntity.java
package com.openvalue.boardgameratings.service.rating;

import lombok.*;

import javax.persistence.*;

import static javax.persistence.GenerationType.IDENTITY;

@AllArgsConstructor
@NoArgsConstructor(access = AccessLevel.PACKAGE)
@Entity
@Getter
class RateEntity {

    @Id
    @GeneratedValue(strategy = IDENTITY)
    private Long id;

    @Column(name = "boardGameName", nullable = false)
    private String boardGameName;


    @Column(name = "rate", nullable = false)
    private Double rate;
}

File: src/main/java/com/openvalue/boardgameratings/service/rating/RateEntity.kt
package com.openvalue.boardgameratings.service.rating

import javax.persistence.*


@Entity
data class RateEntity (

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    val id: Long?,

    @Column(name = "boardGameName", nullable = false)
    val boardGameName: String,


    @Column(name = "rate", nullable = false)
    val rate: Double

)